lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,349 @@
+/* global Symbol */
```

Comment:
I got it from a co-workers fork here: https://github.com/sophypal/ember-cli-code-coverage. @sophypal was this copied from *isparta*?